manyfews/calculations/models.py

```diff
@@ -37,3 +37,18 @@ class InitialCondition(models.Model):
     storage_level = models.FloatField()
     slow_flow_rate = models.FloatField()
     fast_flow_rate = models.FloatField()
+
+
+class RainAndEvapotranspiration(models.Model):
+    date = models.DateTimeField()
+    location = models.PointField(default=Point(0, 0))
+    forecast_hour = models.IntegerField(default=int(0))
+    rain_fall = models.FloatField()
+    potential_evapotranspiration = models.FloatField()
+
+
+class PotentialRiverFlows(models.Model):
+    date = models.DateTimeField()
+    location = models.PointField(default=Point(0, 0))
+    forecast_hour = models.IntegerField(default=int(0))
```

Comment:
Can I check what `date` and `forecast_hour` mean? Is the `date` the start date of the prediction, and `forecast_hour` the offset from that date?
I think it might be better to store `prediction_date` and `forecast_time` as DateTime fields, assuming we need to keep both the time we made this prediction and the time for the flow?

Reply:
accepted and modified.